### DISCUSSION TITLE
Multiple sites on multiple ports

### DISCUSSION THREAD

**Discussion by mikae1** [static-web-server/static-web-server#293] [Q&A] [answered]
I want host multiple websites on multiple domains using Cloudflare Tunnel. I guess I have to publish them om different ports. Can I do this with the a SWS binary or a single Docker image?

**Comment by joseluisq** [comment, marked as answer]
No, SWS can not host multiple websites on multiple ports like nginx does. You have to do it per SWS instance per domain.

But SWS provides [Virtual Hosting](https://static-web-server.net/features/virtual-hosting/) so you could serve files from different root directories depending on the `Host` header of the request using the same SWS instance.
